prusti-specs/src/lib.rs

```diff
@@ -459,20 +456,23 @@ pub fn extern_spec(_attr: TokenStream, tokens:TokenStream) -> TokenStream {
             };
 
             let rewritten_item = handle_result!(
-                extern_spec_rewriter::rewrite_impl(&mut item_impl, Box::from(struct_ty))
+                extern_spec_rewriter::impls::rewrite_impl(&mut item_impl, Box::from(struct_ty))
             );
 
             quote_spanned! {item_span=>
                 #new_struct
                 #rewritten_item
             }
         }
+        syn::Item::Trait(item_trait) => {
+            handle_result!(extern_spec_rewriter::traits::handle_extern_spec_trait(&item_trait))
```

Comment:
I wonder if the other `syn::Item::*` bits can have a corresponding `handle_*` method so this method becomes more readable.